multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -833,6 +838,52 @@ def fit(
 
         return self
 
+    def _split_train_tuning(
+        self, data: pd.DataFrame, holdout_frac: float = None, random_state: int = 0
+    ) -> (pd.DataFrame, pd.DataFrame):
+        """
+        Splits `data` into `train_data` and `tuning_data`.
+        If the problem_type is one of ['binary', 'multiclass']:
+            The split will be done with stratification on the label column.
+            Will guarantee at least 1 sample of every class in `data` will be present in `train_data`.
+                If only 1 sample of a class exists, it will always be put in `train_data` and not `tuning_data`.
+
+        Parameters
+        ----------
+        data : pd.DataFrame
+            The data to be split
+        holdout_frac : float, default = None
+            The ratio of data to use as validation.
+            If 0.2, 20% of the data will be used for validation, and 80% for training.
+            If None, the ratio is automatically determined,
+            ranging from 0.2 for small row count to 0.01 for large row count.
```

Comment:
Are we changing train-test split ratios logic to match tabular here?

If we are changing, this might not be the right thing to do for text models. Generally you’ll need more text rows for validation than in tabular models. 

Specifically, cases like when you have 100k rows, giving 0.01 of that for validation might not enough information for stable validation. On top of that, generally we don't train multiple multi-modal models unlike the defaults in tabular top presets.

Reply:
It is not changed. It was already matching Tabular. Also, 100k will not have 0.01, it will have 0.025 (2500).